docs/UserGuide.md

```diff
@@ -18,85 +18,107 @@ title: User Guide
 
 --------------------------------------------------------------------------------------------------------------------
 
-## 2. About
+## 2. How to use this User Guide
 
-### 2.1 Using this User Guide
+Welcome to *_MyInsuRec_ User Guide*!
 
-Welcome to *MyInsuRec User Guide*!
+Our team would like to thank you for choosing to use _MyInsuRec_! <br>
 
-This guide explains how you can use *MyInsuRec* to organize and manage your clients, meetings and products. This guide has been structured for users to quickly find and understand what they need.
+In order to smoothen the learning curve for you,
+this guide would allow you to effectively learn how to use _MyInsuRec_.
 
-Click on the links below to quickly get to what you need.
+Before we go on further, look out for these symbols in the User Guide as they provide some important message you might want to know!
```

Comment:
should the user guide be lower case?

Reply:
`Before we proceed, look out for these symbols in the user guide as they provide some important messages you might want to be aware of!`

Reply:
Yea will change it